### ISSUE TITLE
codemirror modes importing tests rather than code.

### ISSUE THREAD

**Issue by dmkolobov** [cljsjs/packages#233]
Using `[cljsjs/codemirror "5.6.0-0"]`, when I include CodeMirror and some modes like so:

```clojure
(my-app
  (:require cljsjs.codemirror
            cljsjs.codemirror.mode.css
            cljsjs.codemirror.mode.xml
            cljsjs.codemirror.mode.htmlmixed))
```

I get `Uncaught ReferenceError: test is not defined` in `xml.inc.js` and `css.inc.js`. Upon inspection, it seems that the files are tests for each the modes, rather than the mode JS. What could be causing this issue?

**Comment by crisptrutski** [comment]
@dmkolobov thanks for reporting this issue, but I'm afraid that I've had trouble reproducing it.. Have looked inside the JAR at the `deps.clj` and `js` files, and everything looks correct (no test code). Created a test project also, and it compiled without issue.

Perhaps the package was republished, and your copy is stale? You could try deleting `~/.m2/repository/cljsjs/codemirror`, or compiling your project on another machine. If the problem is still reproducible, perhaps provide an example repo?

**Comment by crisptrutski** [comment]
This should be fixed by #250
